dist/stat_data.py

```diff
@@ -513,6 +513,10 @@ def __init__(self, name, desc, flags=''):
     TxnStat('txn_pinned_snapshot_range', 'transaction range of IDs currently pinned by named snapshots', 'no_clear,no_scale'),
     TxnStat('txn_pinned_timestamp', 'transaction range of timestamps currently pinned', 'no_clear,no_scale'),
     TxnStat('txn_pinned_timestamp_oldest', 'transaction range of timestamps pinned by the oldest timestamp', 'no_clear,no_scale'),
+    TxnStat('txn_prepare', 'transactions prepared'),
+    TxnStat('txn_prepare_active', 'active prepared transactions'),
+    TxnStat('txn_prepare_commit', 'prepare transactions committed'),
+    TxnStat('txn_prepare_rollback', 'prepare transactions rolled back'),
```

Comment:
It would be good if the text always has a common prefix so they sort together. Probably `prepared transactions XXX`.

Specifically I'd like:
`prepared transactions currently active`